### ISSUE TITLE
HA-Firemote.js missing

### ISSUE THREAD

**Issue by dvanroy** [PRProd/HA-Firemote#67]
### Describe the bug

Not exactly a bug but all links to that js are broken

### To Reproduce

Navigate to https://github.com/PRProd/HA-Firemote/blob/main/HA-Firemote.js

### Expected Behavior

allow user to download or install firemote on their HA instance

### Which Type of Device Do You Own?

Not sure, but not relevant

**Comment by PRProd** [comment]
Thanks @dvanroy!  I had to restructure a little bit (#65) and it looks like I neglected to update the README.md file.  This will be updated in the next commit/release.

In the meantime, the path is now https://github.com/PRProd/HA-Firemote/blob/main/dist/  You will need to install the contents of that directory (not just the .js file) into a web accessible Home Assistant location.  (likely /config/www)
